src/common/utils/setup-helpers.js

```diff
@@ -254,10 +244,21 @@ function setLogOptions(options: KPOptionsObject): void {
  * @returns {string} - value of the query string param
  */
 function getUrlParameter(name: string) {
-  name = name.replace(/[[]/, '\\[').replace(/[\]]/, '\\]');
-  const regex = new RegExp('[\\?&]' + name + '=([^&#]*)');
-  const results = regex.exec(location.search);
-  return results === null ? '' : decodeURIComponent(results[1].replace(/\+/g, ' '));
+  const getUrlParamPolyfill = () => {
+    name = name.replace(/[[]/, '\\[').replace(/[\]]/, '\\]');
+    const regex = new RegExp('[\\?&]' + name + '=([^&#]*)');
+    const results = regex.exec(location.search);
+    const isExist = location.search.indexOf(name) > -1;
+    return results === null ? (isExist ? '' : results) : decodeURIComponent(results[1].replace(/\+/g, ' '));
```

Comment:
```suggestion
    return results === null ? (isExist ? 'true' : null) : decodeURIComponent(results[1].replace(/\+/g, ' '));
```
this is clearer. 
also can you add tests for this please?

Reply:
I set empty string because of that is the response from native URLSearchParams